test/unit/test_job.py

```diff
@@ -261,275 +260,6 @@ def test_check_script(self):
-                                        tailer.flush()
-                                else:
-                                    # clear the content of the project file
-                                    for file in os.listdir(Path(temp_dir, f'{expid}/proj/project_files/')):
-                                        os.remove(Path(temp_dir, f'{expid}/proj/project_files/{file}'))
-
-                            # configuration file
-
-                            with open(Path(temp_dir, f'{expid}/conf/configuration.yml'), 'w+') as configuration:
-                                configuration.write(dedent(f'''\
-DEFAULT:
-    EXPID: {expid}
-    HPCARCH: local
-JOBS:
-    A:
-        FILE: a
-        TYPE: {script_type if script_type != "Rscript" else "R"}
-        PLATFORM: local
-        RUNNING: once
-        EXTENDED_HEADER_PATH: {header_file_name}
-        EXTENDED_TAILER_PATH: {tailer_file_name}
-PLATFORMS:
-    test:
-        TYPE: slurm
-        HOST: localhost
-        PROJECT: abc
-        QUEUE: debug
-        USER: me
-        SCRATCH_DIR: /anything/
-        ADD_PROJECT_TO_HOST: False
-        MAX_WALLCLOCK: '00:55'
-        TEMP_DIR: ''
-CONFIG:
-    RETRIALS: 0
-                                '''))
-
-                                configuration.flush()
-
-                            mocked_basic_config = FakeBasicConfig
-                            mocked_basic_config.read = MagicMock()
-
-                            mocked_basic_config.LOCAL_ROOT_DIR = str(temp_dir)
-                            mocked_basic_config.STRUCTURES_DIR = '/dummy/structures/dir'
-
-                            mocked_global_basic_config.LOCAL_ROOT_DIR.return_value = str(temp_dir)
-
-                            config = AutosubmitConfig(expid, basic_config=mocked_basic_config, parser_factory=YAMLParserFactory())
-                            config.reload(True)
-
-                            # act
-
-                            parameters = config.load_parameters()
-                            joblist_persistence = JobListPersistencePkl()
-
-                            job_list_obj = JobList(expid, mocked_basic_config, YAMLParserFactory(),joblist_persistence, config)
-
-                            job_list_obj.generate(
-                                as_conf=config,
-                                date_list=[],
-                                member_list=[],
-                                num_chunks=1,
-                                chunk_ini=1,
-                                parameters=parameters,
-                                date_format='M',
-                                default_retrials=config.get_retrials(),
-                                default_job_type=config.get_default_job_type(),
-                                wrapper_jobs={},
-                                new=True,
-                                run_only_members=config.get_member_list(run_only=True),
-                                show_log=True,
```

Comment:
The duplicated valid `test_header_tailer()` function have a `create=True` in `job_list_obj.generate()` arguments